classes/mail/SubmissionMailTemplate.inc.php

```diff
@@ -51,10 +51,10 @@ function assignParams($paramArray = array()) {
 		$request = Application::get()->getRequest();
 		parent::assignParams(array_merge(
 			array(
-				'submissionTitle' => strip_tags($submission->getLocalizedFullTitle()),
+				'submissionTitle' => htmlspecialchars($submission->getLocalizedFullTitle()),
```

Comment:
I think this will break places where some people write HTML into the submission title, and then customize their theme to show it correctly... Maybe it's better safe than sorry in emails, since an untrusted user could register and submit a dodgy title, but probably not get it published.

Reply:
(Same as: https://github.com/pkp/pkp-lib/pull/7858#discussion_r856476247)